js/index.js

```diff
@@ -0,0 +1,38 @@
+function getRandomInt(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+let dice = {
+	value: 1,
+	roll: () => {
+		this.value = getRandomInt(1, 6);
+		return this.value;
+
+	}
+};
+
+let dice2 = {
+	value: 1,
+	roll: () => {
+		this.value = getRandomInt(1, 6);
+		return this.value;
+
+	}
+};
+function printimage(image, image1){
+	let containar = document.querySelector("#dice1");
+	let containar1 = document.querySelector("#dice2");
+
+	containar.style["background-image"]=`url(/images/inverted-dice-${image}.svg)`;
```

Comment:
Good job! Switching the actual background image on the dice `div` is a great way to make sure it displays the correct image.